scripts/auto-rebase/rebase.sh

```diff
@@ -1094,7 +1094,20 @@ rebase_to() {
     amd64_date="$(cat ${STAGING_DIR}/release_amd64.json | jq -r .config.created | cut -f1 -dT)"
     arm64_date="$(cat ${STAGING_DIR}/release_arm64.json | jq -r .config.created | cut -f1 -dT)"
 
-    rebase_branch="rebase-${ver_stream}_amd64-${amd64_date}_arm64-${arm64_date}"
+    local onto_branch=""
+    # PULL_BASE_REF is expected in CI environments and should identify the active branch.
+    if [[ "${PULL_BASE_REF:-}" == "" ]]; then
+        >&2 echo 'Warning: env var PULL_BASE_REF not found or empty, falling back to local active branch.'
+        onto_branch=$(git branch rev-parse --abbrev-ref HEAD)
```

Comment:
```suggestion
        onto_branch=$(git rev-parse --abbrev-ref HEAD)
```

This command failed for me locally, but removing the "branch" part gave me what looks like a valid branch name.

It would be good to show the onto_branch value after it is computed, to go with the warning on line 1100.

Reply:
Thanks, still running the script in local testing